### ISSUE TITLE
Failing example

### ISSUE THREAD

**Issue by kie0tauB** [pyfisch/cbor#1]
serde + bincode serialization and deserialization work for the attached file, but serde_cbor fails at deserialization with the following error message:

error: "trailing bytes" at byte position 11

[a38348c35f111e17947ffbfe04f4f29252e68aa5b43220860d403b58d1261f15dad7cbe16f8f0b66be392f116ce7ff163fd8b12a6498928aedbe47872408329e.txt](https://github.com/pyfisch/cbor/files/52827/a38348c35f111e17947ffbfe04f4f29252e68aa5b43220860d403b58d1261f15dad7cbe16f8f0b66be392f116ce7ff163fd8b12a6498928aedbe47872408329e.txt)

**Comment by pyfisch** [comment]
Bincode and CBOR are different formats. A deserializer for one format will not work the other format. What data does the example file contain and how was it created?

**Comment by kie0tauB** [comment]
Yes, bincode and cbor are certainly different, but all I'm saying is, while serde works fine with bincode, it fails with cbor, so it's not a serde issue.

**Comment by pyfisch** [comment]
The file you provided is indeed a valid CBOR file. I added a test case to parse the provided file: https://github.com/pyfisch/cbor/commit/9a06446633845b17c3b5abdfe142d397dbff89a4 And the file is successfully parsed as a CBOR value: https://travis-ci.org/pyfisch/cbor/builds/107568165

Maybe you do not try to parse this exact file or you use an outdated version of this crate or serde?

Note: I just updated the crates.io version of CBOR. Maybe it did not work with the old version.
